### ISSUE TITLE
Copy-Item Example 6 and 7

### ISSUE THREAD

**Issue by samsinsane** [MicrosoftDocs/PowerShell-Docs#2230]
Issue Details
-----------------
Examples 6 and 7 do not appear to work as described.

Example 6:
 - When destination doesn't exist, nothing happens.
 - When destination does exist, `C:\Folder002_Copy\Folder002\` is created but no files are copied.

Example 7:
 - When destination doesn't exist:
   - PowerShell 5.0, nothing happens.
   - PowerShell 5.1, `C:\Folder002_Copy\` is created with the contents of `D:\Folder002\`.
 - When destination does exist, `C:\Folder002_Copy\Folder002\` is created with the contents of `D:\Folder002\`.

This issue could exist in other versions of PowerShell, but I only have access to `5.0.10240.17770` and `5.1.14393.693`. For what it's worth, the destination server is running `5.1.14393.1770`.

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [ ] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document
- [x] The documented feature was tested in selected version of PowerShell

**Comment by sdwheeler** [comment]
@joeyaiello Is this provider specific behavior?

**Comment by joeyaiello** [comment]
@Francisco-Gamino might have historical context here, I'll follow up with him offline

**Comment by samsinsane** [comment]
@sdwheeler I'm a little confused by why this was closed, what I commented above doesn't align with what the documentation was updated to say - were these bugs in PowerShell that were fixed instead or something?

**Comment by sdwheeler** [comment]
@samsinsane The docs were updated to correctly describe the behavior. You can see the change made in the PR. The change will go live to the docs website later today.

**Comment by samsinsane** [comment]
> The docs were updated to correctly describe the behavior. You can see the change made in the PR. The change will go live to the docs website later today. Is there some issue with the changes? Can you be specific?

The changes for PowerShell 5.1 example 6 contain the following:
> The operation creates the `Folder002_Copy` folder if it doesn't already exist.

However, my original comment indicates that this doesn't happen. Unfortunately, I don't have access to the setup I had two years ago, so I can't provide any further information about this.

**Comment by sdwheeler** [comment]
@chasewilson Can you retest this?

**Comment by chasewilson** [comment]
Hi @samsinsane thank you for the feedback. I've been testing the examples to verify results. 

I've been using the latest version of PowerShell 5.1: `5.1.18362.145`. PowerShell 5.0 is no longer supported so i did not test that version. 

My test results for both examples work in the expected way which is as follows:

Destination did not exist - Folder was created but contents were not copied(The section for this has been fixed to show that contents will not be copied). Also, when the item is copied, its is  renamed to reflect the `-Destination`  param value. E.g. copying `Folder002` to `-Destination C:\Folder002_Copy` renames the folder to `Folder002_Copy` . This is the same expected result if you use the `-Recurse` switch. The folder is created with the name `Folder002_Copy` and the contents are copied. 
    If you define the `-Destination` with a path that doesn't exist ending with the same name as a folder it will create the full path. E.g. 
```powershell
Copy-Item 'C:\Folder002' -Destination 'C:\NewTempFolder\Folder002' -ToSession $session
```
This creates `NewTempFolder` with the `Folder 002` Copied inside it. If the `-Recurse` switch is added, the contents of `Folder002` are copied as well.

Destination does exist - Since the destination is a preexisting Directory, it copies the item without renaming it to the `Folder002_Copy` and just copies it inside the existing directory. Same for adding the `-Recurse` switch .

I have tested across PowerShell 5.1, 6, and 7. The result is the same. does this clear it up?
